server/command.go

```diff
@@ -31,9 +31,41 @@ const COMMAND_HELP = `* |/gitlab connect| - Connect your Mattermost account to y
 * |/gitlab me| - Display the connected GitLab account
 * |/gitlab settings [setting] [value]| - Update your user settings
   * |setting| can be "notifications" or "reminders"
-  * |value| can be "on" or "off"`
-
+  * |value| can be "on" or "off"
+* |/gitlab webhook list [owner]/repo| - Will list associated group or project hooks.
+* |/gitlab webhook add owner[/repo] [options] [url] [token]|
+  * |options| is a comma-delimited list of one or more the following:
+	 * |*| - or missing defaults to all with SSL verification enabled
+	 * *noSSL - all triggers with SSL verification not enabled.
+	 * PushEvents
+	 * TagPushEvents 
+	 * Comments 
+	 * ConfidentialComments 
+	 * IssuesEvents
+	 * ConfidentialIssuesEvents 
+	 * MergeRequestsEvents 
+	 * JobEvents 
+	 * PipelineEvents 
+	 * WikiPageEvents
+	 * SSLverification
+  * |url| is the URL that will be called when triggered. Defaults to this plugins URL
+  * |token| Secret token. Defaults to secrete token used in plugin's settings.
```

Comment:
Similar question here

Reply:
creating a webhook for your CI/CD tooling or another chat or notification service